jupyterlab_nodeeditor/yggdrasil_support.py

```diff
@@ -1,9 +1,37 @@
 import uuid
-
 import yaml
 
 from .node_editor import NodeEditor
 
+from yggdrasil.examples import yamls as ex_yamls
+import jupyterlab_nodeeditor as jlne
+import yggdrasil.yamlfile
+import re
+
+photosynthesis_model = {}
+with open(ex_yamls['fakeplant']['python'], "r") as test_model:
+    photosynthesis_model = yaml.safe_load(test_model)['model']
```

Comment:
```suggestion
with open(ex_yamls['fakeplant']['python'], "r") as test_model:
    photosynthesis_model = yaml.safe_load(test_model)['model']
```

We don't need to declare it since we're redefining it inside the `with`